src/qibolab/compilers/compiler.py

```diff
@@ -129,12 +129,19 @@ def compile(self, circuit, platform):
 
         measurement_map = {}
         # process circuit gates
+        delays = {qubit: 0 for qubit in platform.qubits.keys()}
```

Comment:
A shortcut for this would be
```suggestion
        delays = defaultdict(int)
```
This assumes that delays are integers, which I believe is the case around qibolab currently.